applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -95,10 +109,13 @@ content_types_accepted_by_verb(Context, ?HTTP_POST) ->
 %%--------------------------------------------------------------------
 -spec validate(cb_context:context()) -> cb_context:context().
 -spec validate(cb_context:context(), path_token()) -> cb_context:context().
+-spec validate(cb_context:context(), path_token(), path_token()) -> cb_context:context().
 validate(Context) ->
     validate_rates(Context, cb_context:req_verb(Context)).
 validate(Context, Id) ->
     validate_rate(Context, Id, cb_context:req_verb(Context)).
+validate(Context, ?NUMBER, Phonenumber) ->
+    rate_for_number(Context, Phonenumber).
```

Comment:
Hi Luis.
Not sure what is the best way to validate number.
Looks like wnm_util doesn't provides anything special.
So I just clean up string from anything but digits and check if it not very few of it.